jenkins/dockerized-e2e-runner.sh

```diff
@@ -85,7 +85,7 @@ kill-leaked-container() {
 trap "kill-leaked-container ${CONTAINER_NAME}" EXIT
 
 echo "Starting..."
-docker run --rm -i \
+docker run --rm \
```

Comment:
yes, a) jobs shouldn't be interactive and b) jobs hang for an absurd amount of time after completing when I run this from my workstation and this is present (I believe this doesn't happen on jenkins because there is no tty)